daemon/src/db.rs

```diff
@@ -283,6 +283,56 @@ pub async fn load_all_cfd_ids(conn: &mut PoolConnection<Sqlite>) -> Result<Vec<O
     Ok(ids)
 }
 
+/// Loads cfd that are not final yet
+///
+/// Finality can happen by:
+/// 1. Recording the confirmation of a payout (spend) transaction on the blockchain
+///     Cases: Collaborative settlement, CET, Refund
+/// 2. The CFD failing early meaning it becomes irrelevant for processing
+///     Cases: Setup failed, Taker's take order rejected
+pub async fn load_non_final_cfd_ids(conn: &mut PoolConnection<Sqlite>) -> Result<Vec<OrderId>> {
+    let (collaborative_settlement_confirmed, _) =
+        CfdEvent::CollaborativeSettlementConfirmed.to_json();
+    let (cet_confirmed, _) = CfdEvent::CetConfirmed.to_json();
+    let (refund_confirmed, _) = CfdEvent::RefundConfirmed.to_json();
+    let (setup_failed, _) = CfdEvent::ContractSetupFailed.to_json();
+    let (rejected, _) = CfdEvent::OfferRejected.to_json();
```

Comment:
I think I'd prefer if we have these names as constants somewhere and reference them directly instead of computing this every time. We did this already for the caching, have a look at that!

Reply:
I think it is super cheap here because non of these events actually has data, but happy to use constants, that's even better.